salt/modules/dsc.py

```diff
@@ -194,3 +194,69 @@ def remove(name):
     cmd = 'Uninstall-Module {0}'.format(name)
     no_ret = _pshell(cmd)
     return name not in list_modules()
+
+
+def run_config(path):
+    # Run the DSC Configuration
+    cmd = '$job = Start-DscConfiguration -Path {0};'.format(path)
+    cmd += 'Do{ } While ($job.State -notin \'Completed\', \'Failed\'); ' \
+           'return $job.State'
+    ret = __salt__['cmd.run'](cmd, shell='powershell', python_shell=True)
```

Comment:
With `python_shell=True` are you certain this is not vulnerable to shell injection attacks? We should ensure that a maliciously crafted argument could not be inserted here.